lib/network/modules/NodesHandler.js

```diff
@@ -294,6 +298,24 @@ class NodesHandler {
         nodes[id] = node;
       }
     }
+
+    if (!dataChanged && oldData !== undefined) {
```

Comment:
Well, I like explicit testing because I'm paranoid with respect to conditions in the code. But I'll admit that I can go over the top sometimes.